CSS/cart.css

```diff
@@ -0,0 +1,153 @@
+@import url(/CSS/style.css);
+@import url(/CSS/wishlist.css);
+
+.cart_card_container,
+.cart_quantity,
+.li_flex,
+.cart_container,
+.cart_price,
+.cart_btns {
+    display: flex;
+    flex-wrap: wrap;
+}
+
+.price_title,
+.btn_cart {
+    padding: 1rem;
+}
+
+.cart_remove_btn,
+.place_order,
+.cart_wishlist_btn:hover {
+    background-color: var(--lightblue-color);
+    color: var(--pureWhite-color);
+}
+
+.cart_card_container,
+.price_detailes_container,
+.qunt_btn,
+.cart_wishlist_btn {
+    background-color: var(--pureWhite-color);
+}
+
+.btn_cart,
+.cart_items,
+.input_num,
+.qunt_btn {
+    font-size: 1rem;
+}
+
+.img_container {
+    border-right: 1px solid var(--boder-color);
+}
+
+.cart_quantity,
+.cart_price {
+    padding-top: 0.5rem;
+}
+
+.cart_off,
+.cut_price {
+    padding-top: 1rem;
+}
+
+.cart_card_container,
+.price_detailes_container {
+    margin-top: 3rem;
+    box-shadow: 2px 2px 2px 2px var(--boder-color);
+}
+
+.cart_container {
+    margin-top: 2rem;
+    margin-bottom: 2rem;
```

Comment:
Use shorthand for this, margin : 2rem 0;

Reply:
okay